content/_data/changelogs/weekly.yml

```diff
@@ -7562,6 +7562,78 @@
 
   # pull: 4708 (PR title: Fix up Protector class)
 
+- version: '2.237'
+  date: 2020-05-18
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4712
+    issue: 61920
+    authors:
+    - MRamonLeon
+    message: |-
+      Internal: Avoid populating logs with missing classes telemetry warnings (regression in TODO).
+  - type: bug
+    category: bug
+    pull: 4718
+    issue: 62231
+    authors:
+    - MRamonLeon
+    message: |-
+      Internal: Avoid an error when reporting errors because of field unmarshalling to stop loading a plugin.
+  - type: bug
+    category: bug
+    pull: 4711
+    issue: 60118
+    authors:
+    - oleg-nenashev
+    message: |-
+      Ensure that UserLanguages telemetry initializer always runs after extensions are augmented.
+  - type: bug
+    category: bug
+    pull: 4684
+    issue: 61956
+    authors:
+    - calvinpark
+    message: |-
+      Check folder name for valid characters.
+  - type: rfe
+    category: developer
+    pull: 4725
+    authors:
+    - jvz
+    message: |-
+      Developer: Update Apache Ant to 1.10.8.
+  - type: rfe
+    category: developer
+    pull: 4656
+    authors:
+    - jvz
+    references:
+      - pull: 4656
+      - url: https://tomcat.apache.org/taglibs/CHANGES.txt
+        title: Changelog up to 1.2.5
+      - url: https://github.com/eclipse-ee4j/jstl-api/compare/1.2.3-RELEASE..1.2.7-RELEASE
+        title: Diff of 1.2.3 to 1.2.7
+      - url: https://github.com/javaee/jstl-api/compare/javax.servlet.jsp.jstl-1.2.1..javax.servlet.jsp.jstl-1.2.3
+        title: Diff of 1.2.1 to 1.2.3
+    message: |-
+      Developer: Update JSTL API to 1.2.7.
+  - type: rfe
+    category: developer
+    pull: 4715
+    authors:
+    - jglick
+    message: |-
+      Developer: Deprecate <code>jenkins.model.Configuration</code> in the Java API.
+
+  # pull: 4689 (PR title: Minor pom cleanup and switched assertions to junit5)
+  # pull: 4717 (PR title: Adapt to JUnit 4.13's deprecation of org.junit.Assert.assertThat)
+  # pull: 4720 (PR title: System read code simplification)
+  # pull: 4722 (PR title: Taglib descriptions)
+  # pull: 4723 (PR title: Reduce heading for repeatable)
```

Comment:
Maybe could be a good addition as `rfe` to highlight the UX changes

Reply:
I did not include this suggestion in the merged changelog.  Let me know if you feel strongly that it needs to be in the changelog